bootloader/src/pyi_pythonlib.c

```diff
@@ -88,12 +89,12 @@ pyi_pylib_load(ARCHIVE_STATUS *status)
       strncpy(dllname, status->cookie.pylibname, 64);
     }
 #else
-    len = 0;
     strncpy(dllname, status->cookie.pylibname, 64);
```

Comment:
how about using this at these 2 places:
```
len = snprintf(dllname, 64, "%s", status->cookie.pylibname);
```

It's of course a rather expensive call (but that does not matter), but at least it would set `len` to the correct value (with the current code, len == 0 and len is actually used below).

Reply:
Agreed, as we use `snprintf` a few lines above anyway. So this would even give a consistent code.